scrapypuppeteer/middleware.py

```diff
@@ -107,12 +96,20 @@ def from_crawler(cls, crawler):
             include_headers = cls.DEFAULT_INCLUDE_HEADERS
         include_meta = crawler.settings.getbool(cls.SERVICE_META_SETTING, False)
 
-        if local_mode:
-            browser_manager = LocalBrowserManager()
-        else:
+        execution_method = crawler.settings.get(
```

Comment:
Add a new chapter in documentation `execution method` or smth like this.

Reply:
Done